app/models/etsource/loader.rb

```diff
@@ -46,8 +46,11 @@ def presets
 
     def area_attributes(area_code)
       @area_attributes[area_code] ||= begin
-        area_attr = Atlas::Dataset.find(area_code).to_hash
+        area = Atlas::Dataset.find(area_code)
+        area_attr = area.to_hash
+        area_attr[:is_derived] = true if area.is_a?(Atlas::Dataset::DerivedDataset)
```

Comment:
I agree with @grdw. Also, I don't think it's right to have the `is_` prefix; such prefixes are normally omitted in Ruby, especially in predicate methods.

For example:

* `thing.name`, not `thing.get_name`
* `thing.name=`, not `thing.set_name`
* `thing.empty?` not `thing.is_blank?`
* **`thing.derived?` not `thing.is_derived?`**

Otherwise, once the tests pass I think this can be merged.